RestSharp/Http.cs

```diff
@@ -197,6 +197,15 @@ public Encoding Encoding
         /// </summary>
         public bool PreAuthenticate { get; set; }
 
+        /// <summary>
+        /// Flag to reuse same connection in the HttpWebRequest
+        /// </summary>
+        public bool UnsafeAuthenticatedConnectionSharing { get; set; }
+
+        /// <summary>
+        /// Flag to send authorisation header with the HttpWebRequest
+        /// </summary>
+        public bool UseUnsafeConnection { get; set; }
```

Comment:
unused?

Reply:
@KOTRET could you elaborate?

Reply:
@alexeyzimarev prop `UnsafeAuthenticatedConnectionSharing` (Line 203) is definitely in use, but i cannot find usage of the prop `UseUnsafeConnection` (line 208) , which is also added with this commit.

Reply:
This is a public API in a library. "Unused" in this context makes little sense. Most of the public API methods are "unused" inside the library, they are public because people who download the library will use it.
I agree in a sense that if it is "unused" - it means there are no tests for it. This actually is a problem.

Reply:
@KOTRET I see what you mean. It is indeed added without any usage.